### PR TITLE
Add README, LICENSE and OpenAPI docs

### PR DESCRIPTION
**Pull request by pferreir**
Just to improve the situation with regards to project documentation. I also bumped the dependencies that have new versions.

Example API docs: https://pferreir.github.io/ursh/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,9 +17,21 @@ install:
   - pip install flake8
   - pip install isort
   - pip install -e .
+  - npm install redoc-cli
```
Comment (ThiefMaster) [inline .travis.yml]: You don't need to `npm install` packages you want to run with `npx`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+First, install the package with support for testing:
+
+```sh
+pip install -e .[testing]
+```
+
+Then you can run the tests:
+
+```sh
+py.test .
```
Comment (ThiefMaster) [inline README.md]: `pytest` is enough (and `py.test` is deprecated)

File: setup.py [hunk 3]
```diff
@@ -35,10 +35,12 @@ def get_requirements():
     packages=find_packages(),
     install_requires=get_requirements(),
     setup_requires=[] + pytest_runner,
```
Comment (ThiefMaster) [inline setup.py]: I think we can get rid of the pytest stuff in this file - nobody really uses `python setup.py test` anymore anyway

File: ursh/core/app.py [hunk 4]
```diff
@@ -112,7 +112,7 @@ def _register_docs(app):
     from ursh.schemas import TokenSchema, URLSchema
     with app.app_context():
         spec = APISpec(
-            openapi_version='2.0',
+            openapi_version='3.0.2',
```
Comment (ThiefMaster) [inline ursh/core/app.py]: This works? I thought I tried updating the openapi version but then got errors when trying to access e.g. `/swagger-ui/` about the openapi specs we have not being valid valid with that version.

Comment (ThiefMaster) [reply ursh/core/app.py]: OK looking at the Travis output I think it doesn't work ;)

Comment (pferreir) [reply ursh/core/app.py]: Seems to be enough to generate the JSON...

File: .travis.yml (hunk 1 above)
Comment (pferreir) [reply .travis.yml]: Yeah, right.

File: ursh/core/app.py (hunk 4 above)
Comment (pferreir) [reply ursh/core/app.py]: It looks as if the JSON that is generated is somehow wrong. I'll investigate.

Comment (pferreir) [reply ursh/core/app.py]: Weird. It works OK locally 😲

Comment (pferreir) [reply ursh/core/app.py]: OK, easy fix!

**Pull request merged**